android/lib/src/main/java/com/swmansion/gesturehandler/FlingGestureHandler.java

```diff
@@ -0,0 +1,97 @@
+package com.swmansion.gesturehandler;
+
+import android.os.Handler;
+import android.view.MotionEvent;
+
+public class FlingGestureHandler extends GestureHandler<FlingGestureHandler> {
+  private static final long DEFAULT_MAX_DURATION_MS = 800;
+  private static final long DEFAULT_MIN_ACCEPTABLE_DELTA = 90;
+  private static final int DEFAULT_DIRECTION = DIRECTION_RIGHT;
+  private static final int DEFAULT_NUMBER_OF_TOUCHES_REQUIRED = 1;
+
+  private long mMaxDurationMs = DEFAULT_MAX_DURATION_MS;
+  private long mMinAcceptableDelta = DEFAULT_MIN_ACCEPTABLE_DELTA;
+
+  private int direction = DEFAULT_DIRECTION;
+  private int mNumberOfTouchesRequired = DEFAULT_NUMBER_OF_TOUCHES_REQUIRED;
+  private float mStartX, mStartY;
+
+  private Handler mHandler;
+  private int mMaxNumberOfTouchesSimultaneously;
+
+  private final Runnable mFailDelayed = new Runnable() {
+    @Override
+    public void run() {
+      fail();
+    }
+  };
+
+  public void setNumberOfTouchesRequired(int mNumberOfTouchesRequired) {
+    this.mNumberOfTouchesRequired = mNumberOfTouchesRequired;
```

Comment:
Then you will also just write:
`mNumberOfTouchesRequired = numberOfTouchesRequired`

(no `this.`)